tests/basic/sharedv4_test.go

```diff
@@ -113,6 +115,17 @@ var _ = Describe("{Sharedv4Functional}", func() {
 				// `/proc/self/mountinfo`
 				Step(fmt.Sprintf("mark device path as RO %s", ctx.App.Key), func() {
 					setPathToROMode(devicePath, attachedNode)
+
+					// check only mnt.VfsOpts (fs state) is RO
+					mntList, err := mount.GetMounts()
+					Expect(err).NotTo(HaveOccurred())
+					for _, mnt := range mntList {
+						if mnt.Mountpoint != devicePath {
+							continue
+						}
+						Expect(volumeMountRW.MatchString(mnt.Opts)).To(BeTrue())
+						Expect(volumeMountRO.MatchString(mnt.VfsOpts)).To(BeTrue())
```

Comment:
This looks fine. If the same problem happens again, the test will fail earlier which is good. But the test will still fail, yes? How do we make the test not fail?

Reply:
yeah, we are still seeing the same failure in the latest test. I think we might need a more robust way to make sure the mount is set to the expected permissions. I'll add a future task for this.